monitoring/uss_qualifier/scd/data_interfaces.py

```diff
@@ -0,0 +1,31 @@
+from typing import Optional
+from monitoring.monitorlib.typing import ImplicitDict
+from monitoring.monitorlib.scd_automated_testing.scd_injection_api import InjectFlightRequest
+import datetime
+
+class SCDTestIssue(ImplicitDict):
+    ''' A class to hold a message that the test executor can provide to the USS in cases when the USS provides a response that is not same as the expected result. '''
+    timestamp: Optional[str]
```

Comment:
This field is not appropriate for a data structure that will store the output of the simulator because the time the simulator generated the set of `TestInjectionRequiredResult`s is irrelevant.  The time the issue occurred (during test execution by test executor) is definitely relevant and should be included on the SCD Issue, but let's generally try to be architecturally consistent with [existing work](https://github.com/interuss/dss/blob/10f5192c17695566347eba3e52843ce1c48d06dc/monitoring/uss_qualifier/rid/reports.py#L22) and put the SCD Issue in `reports.py` when we are ready to define it (probably not this PR).

Reply:
Thank you for this, I have now removed the timestamp. Re moving to reports.py we can tackle this later but I noticed that it sits in the rid folder, we may need to create a new common report file. I agree that is for later.